app/timetables/models.py

```diff
@@ -175,3 +175,27 @@ def __str__(self):
 
     class Meta:
         unique_together = ('timetable', 'cycle_day', 'meal', 'meal_option')
+
+
+class Event(TimestampMixin):
+    """
+    Model representing a date or range of dates to which a
+    specific action can be assigned.
+    """
```

Comment:
According to [PEP standard](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings), `Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.` Please update the docstring to follow PEP standard.